contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -159,6 +160,25 @@ contract InitializableAbstractStrategy is Initializable, Governable {
         _setPTokenAddress(_asset, _pToken);
     }
 
+    /**
+     * @dev Remove a supported asset by passing its index.
+     *      This method can only be called by the system Governor
+     * @param _assetIndex Index of the asset to be removed
+     */
+    function removePToken(uint256 _assetIndex) external onlyGovernor {
+        require(_assetIndex < assetsMapped.length, "Invalid index");
+        address asset = assetsMapped[_assetIndex];
+        address pToken = assetToPToken[asset];
+
+        if (_assetIndex < assetsMapped.length - 1) {
+            assetsMapped[_assetIndex] = assetsMapped[assetsMapped.length - 1];
+        }
+        delete assetsMapped[assetsMapped.length - 1];
```

Comment:
This won't decrease the array length I believe, also see #374 and TOB-OUSD-012

Reply:
Have changed it to use `.pop()` method